splash/browser_tab.py

```diff
@@ -512,6 +540,28 @@ def _on_url_changed(self, url):
         self.web_page.har.store_redirect(six.text_type(url.toString()))
         self._cancel_timers(self._timers_to_cancel_on_redirect)
 
+    def _on_unsupported_content_finished(self):
+        self.logger.log('Unsupported content finished', min_level=3)
+        if not self._load_finished_after_unsupported_content_ready:
+            # XXX: The unsupported content reply might have finished before the
+            # original loadFinished signal emits. In such cases we do not want
+            # the same signal twice.
+            if not self._unsupported_content_reply.error():
+                self.web_page.mainFrame().loadFinished.emit(True)
+            else:
+                self.web_page.mainFrame().loadFinished.emit(False)
+
+    def _on_unsupported_content(self, reply):
+        self.logger.log('Unsupported content detected', min_level=3)
+        self._is_unsupported_content = True
```

Comment:
this attribute is set per browser tab and browser tab can be used to download multiple resources, if you set this attribute here will it affect other requests going through same webpage? e.g. script doing

```lua
splash:go("url_with_unsupported_content") # this will set _is_unsupported_content to True
splash:go("url with supported content") # will this go through?
```

other thing worth noting is that there could be multiple resources downloaded when rendering so e.g.

```lua
splash:go("some page")
```
may issue 10 requests, first 6 will have supported content, but 7th will have unsupported content, what will happen with resources 7-10? will they be downloaded all right?